test/e2e_node/container_lifecycle_test.go

```diff
@@ -71,23 +71,23 @@ var _ = SIGDescribe(framework.WithNodeConformance(), "Containers Lifecycle", fun
 				InitContainers: []v1.Container{
 					{
 						Name:  init1,
-						Image: busyboxImage,
+						Image: agnhostImage,
```

Comment:
should we also update the ExecCommand function to be OS agnostic? Changing to agnhost here kind of suggests that the test should be working on Windows